### PR TITLE
fix/content-updates

### PR DESCRIPTION
**Pull request by agraebe**
irt #657

### CODE REVIEW

File: _includes/keyphrase.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Use the following table to answer questions about keys/phrases/values you can sh
       </td>
    </tr>
    <tr valign="top">
-      <td><p>Stacks address or STX address</p>
+      <td><p>Stacks address or Stacks address</p>
```
Comment (aulneau) [inline _includes/keyphrase.md:56]: Why was this change made? `Stacks address or Stacks address` seems like a mistake. If possible, I'd not edit this file nor the glossary file.

File: _data/glossary.csv [hunk 2]
```diff
@@ -49,7 +49,7 @@ Secret Recovery Key,<p>Used to access an identity on the Blockstack blockchain.
...
 soft fork,"A soft fork is a proposed update to the software governing the network that results in a post-update network that is compatible with the network as it existed prior to the update, because it restricts the network operations that can be performed after the update."
-Stacks (STX) address,"<p> A STX address is a string of letters and numbers starting with an <code>SP</code> or <code>SM</code>, for example: <code>SM3KJBA4RZ7Z20KD2HBXNSXVPCR1D3CRAV6Q05MKT</code> </p><p> If you created a software-only wallet with the Stacks Wallet software, the wallet has a single STX address which is also sometimes called the <em>wallet address</em>. You access a software wallet with a <em>seed phrase</em>. </p>"
+Stacks (STX) address,"<p> A Stacks address is a string of letters and numbers starting with an <code>SP</code> or <code>SM</code>, for example: <code>SM3KJBA4RZ7Z20KD2HBXNSXVPCR1D3CRAV6Q05MKT</code> </p><p> If you created a software-only wallet with the Stacks Wallet software, the wallet has a single Stacks address which is also sometimes called the <em>wallet address</em>. You access a software wallet with a <em>seed phrase</em>. </p>"
```
Comment (aulneau) [inline _data/glossary.csv:52]: I would revert this change.

**Comment by agraebe** [comment]
@aulneau makes sense. let me revert the STX>Stacks changes in unrelated files

File: _data/glossary.csv (hunk 2 above)
Comment (agraebe) [reply _data/glossary.csv]: done!

File: _includes/keyphrase.md (hunk 1 above)
Comment (agraebe) [reply _includes/keyphrase.md]: done as well
